appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -5177,7 +5177,7 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @Description("")
   String HorizontalArrangementHelpStringComponentPallette();
 
-  @DefaultMessage("Component for displaying images.  The picture to display, and other aspects of the Image\"s appearance, can be specified in the Designer or in the Blocks Editor.")
+  @DefaultMessage("Component for displaying images.  The picture to display, and other aspects of the image\"s appearance, can be specified in the Designer or in the Blocks Editor.")
```

Comment:
As commented below, Image is a noun (name of the component) so the Capital is fine.